safe/gui/tools/multi_buffer_dialog.py

```diff
@@ -0,0 +1,235 @@
+# coding=utf-8
+"""**Volcano Buffer Implementation.**
+
+"""
+
+import logging
+import os
+
+from qgis.core import QgsVectorFileWriter, QgsMapLayerRegistry, QgsVectorLayer
+from qgis.gui import QgsMapLayerProxyModel
+from PyQt4 import QtGui
+from PyQt4.QtCore import pyqtSignature, pyqtSlot
+from PyQt4.QtGui import QFileDialog, QIcon
+from safe.common.version import get_version
+from safe.gis.vector.multi_buffering import multi_buffering
+from safe.gui.tools.help.multi_buffer_help import (
+    multi_buffer_help)
+from safe.messaging import styles
+from safe.utilities.resources import (
+    get_ui_class,
+    resources_path,
+    html_footer,
+    html_header)
+
+INFO_STYLE = styles.BLUE_LEVEL_4_STYLE
+LOGGER = logging.getLogger('InaSAFE')
+FORM_CLASS = get_ui_class('multi_buffer_dialog_base.ui')
+
+
+class MultiBufferDialog(QtGui.QDialog, FORM_CLASS):
+    """Dialog implementation class for the InaSAFE multi buffer tool."""
+
+    def __init__(self, parent=None):
+        """Constructor for the multi buffer dialog.
+
+        :param parent: Parent widget of this dialog
+        :type parent: QWidget
+        """
+        QtGui.QDialog.__init__(self, parent)
+        self.setupUi(self)
+        self.setWindowTitle(self.tr(
+            'InaSAFE %s Multi Buffer Tool' % get_version()))
```

Comment:
+1 to skip the version - it will just result in soon outdated screenshots anyway...